packages/client/wallets/smart-wallet/src/blockchain/wallets/passkey.ts

```diff
@@ -108,6 +108,10 @@ export class PasskeyAccountService {
     }
 
     private mapError(error: any, passkeyName: string) {
+        if (error.message === "Registration not verified") {
```

Comment:
This is pretty brittle XD, but there's literally no other properties the error has, so doing the best we can here.

Reply:
same Q here, there's no const from zerodev/viem that we can point to?
Otherwise if this is the best we can do, then is what it is

Reply:
That's the case unfortunately

Code in the ZD SDK
```TS
if (!registerVerifyResult.verified) {
    throw new Error("Registration not verified")
}
```